### PR TITLE
Name is lie, its embed/attachment

### PR DESCRIPTION
**Pull request by WackyModer**
I made no mistakes :3 (lie)

I don't think the code will break. Also we need an icon.

### CODE REVIEW

**Comment by WackyModer** [comment]
I'm bad at GitHub PR's, so sorry about the repeated commits.

File: public/client.js [hunk 1]
```diff
@@ -514,22 +514,26 @@ function formatSizeUnits(size){
...
 //makes all unrecognized files downloadable
-function unknown_file_dowloadable(attached){
+function unknown_file_downloadable(attached){
     let thegotten = getFileSize(attached)
-    return `<a href='${attached}'>${thegotten[1].split('"')[1]+" ("+thegotten[0]+")"}</a>`
+    return `<div class="downloadable_file_div"><a class="downloadable_file" target='_blank' href='${attached}'>${thegotten[1].split('"')[1]+" ("+thegotten[0]+")"}<img class="embed-dl-img" src='./assets/img/download_icon.svg' width=30></a>`
+}
+
+function adddedlbutton(attache){
+    return `<a style="padding-left: 1rem;" href='${attache}' target="_blank" rel="noreferrer noopener"><img src='./assets/img/download_icon.svg' width=30></a>`;
```
Comment (Vukkyy) [inline public/client.js:523]:
```suggestion
    return `<a style="padding-left: 1rem;" href='${attache}' target="_blank" rel="noreferrer noopener"><i class="fa-solid fa-download"></i></a>`;
```
You don't appear to have included the "download icon" SVG in this commit.
That being said, Quarky (*very legally*) uses Font Awesome, so you can just use their icons instead.

**Review by Vukkyy — CHANGES_REQUESTED** [review]

File: public/client.js (hunk 1 above)
Comment (WackyModer) [reply public/client.js:523]: oopsie

**Comment by WackyModer** [comment]
wait how do I accept your change

**Comment by WackyModer** [comment]
I think I did it

File: public/client.js [hunk 2]
```diff
@@ -514,22 +514,26 @@ function formatSizeUnits(size){
     return giga.toFixed(1) + " GB";
 }
 //makes all unrecognized files downloadable
-function unknown_file_dowloadable(attached){
+function unknown_file_downloadable(attached){
     let thegotten = getFileSize(attached)
-    return `<a href='${attached}'>${thegotten[1].split('"')[1]+" ("+thegotten[0]+")"}</a>`
+    return `<div class="downloadable_file_div"><a class="downloadable_file" target='_blank' href='${attached}'>${thegotten[1].split('"')[1]+" ("+thegotten[0]+")"}<img class="embed-dl-img" src='./assets/img/download_icon.svg' width=30></a>`
```
Comment (Vukkyy) [inline public/client.js:519]:
```suggestion
    return `<div class="downloadable_file_div"><a class="downloadable_file" target='_blank' href='${attached}'>${thegotten[1].split('"')[1]+" ("+thegotten[0]+")"}<i class="fa-solid fa-download"></i></a>`
```
I forgot to do it here too :<

**Review by Vukkyy — CHANGES_REQUESTED** [review]

**Review by Vukkyy — APPROVED** [review]

**Pull request merged**